optax/_src/update_test.py

```diff
@@ -79,5 +79,9 @@ def test_periodic_update(self):
           params_2, new_params, atol=1e-10, rtol=1e-5)
 
 
+  def test_none_argument(self):
```

Comment:
Make it a parameterized test, that is, add `from absl.testing import parameterized` at the top and then write the test as
```
  @parameterized.named_parameters(
      dict(testcase_name='apply_updates', operation=update.apply_updates),
      dict(testcase_name='incremental_update', operation=update.incremental_update),
  )
    def test_none_argument(self, operation)
      x = jnp.array([1., 2., 3.])
      operation(None, x)
  ```

Reply:
Done. However, the functions take a different number of arguments. To make it compatible I've used a lambda expression. Also below for tree_math_test. Let me know if this is fine or if there is a better way to deal with that.

Reply:
That's perfect, thank you!